scripts/ansible/roles/linux/common/vars/ubuntu/bionic.yml

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+---
+llvm_apt_repository: "deb http://apt.llvm.org/{{ ansible_distribution_release }} llvm-toolchain-{{ ansible_distribution_release }}-8 main"
```

Comment:
Is this repo used to install Clang? The recent build has Clang 8.0.1, which fails the validation step as it requires Clang 8.0.0.

So this may need to be reverted to -7 or we update scripts\ansible\roles\linux\openenclave\vars\ubuntu\main.yml to use 8.0.1 as well.

Reply:
Yeah, I'll revert it to -7. I don't see a point going to 8.0.1 since we're looking at updating to clang-10 or 11 anyways.